docs(instructions): (#1516)
modelcontextprotocol/python-sdk: docs/concepts.md

  - Context and sessions
  - Lifecycle and state
 -->

## Server Instructions

> When a server initializes, it can send instructions tot he client explaining how the tools can be used as a collective group, this can be thought of like an instructions manual for the consumer of a given server.

Reviewer: ```suggestion
When a server initializes, it can send instructions to the client explaining how the tools can be used as a collective group, this can be thought of like an instructions manual for the consumer of a given server.
```

Author: whoops thank you good catch
